Document/0x07c-Testing-Cryptography.md

```diff
@@ -8,16 +8,15 @@ Proper cryptographic key management is a common pitfall when designing mobile ap
 
 #### Overview
 
-Choosing strong cryptographic algorithm alone is not enough. Often security of otherwise sound algorithms can be affected if misconfigured. Many previously strong algorithms and their configurations are now considered vulnerable or non-compliant with best practices. It is therefore important to periodically check current best practices and adjust configurations accordingly.  
-
 Many cryptographic algorithms and protocols should not be used because they have been shown to have significant weaknesses or are otherwise insufficient for modern security requirements.
 
+Many previously strong algorithms and their configurations are now considered vulnerable or non-compliant with best practices. It is therefore important to periodically check current best practices and adjust configurations accordingly.  
```

Comment:
This says approximately the same as the sentence before it. Maybe it's best to merge the two paragraphs.